src/test/java/Homework2/Homework2.java

```diff
@@ -0,0 +1,184 @@
+    @Test
+    public void verifySignUpIsEnabled () {
+        MyDriver.launchUrlOnNewWindow("https://www.facebook.com/");
+        String createNewAccountXpath = "//a[text()='Create new account']";
+        By createAccountLocator = By.xpath(createNewAccountXpath);
+        WebElement createAccountButton = MyDriver.getDriver().findElement(createAccountLocator);
+        createAccountButton.click();
+
+        Misc.pause(5);
+
+        // get the locator for first name box and type name
+        String firstNameXpath = "//input[@placeholder='First name']";
+        By firstNameLocator = By.xpath(firstNameXpath);
+        WebElement firstNameBox = MyDriver.getDriver().findElement(firstNameLocator);
+        firstNameBox.sendKeys("joy");
+
+        Misc.pause(3);
+
+        // get the locator for last name box and type name
+        String lastNameXpath = "//input[@name='lastname']";
+        By lastNameLocator = By.xpath(lastNameXpath);
+        WebElement lastNameBox = MyDriver.getDriver().findElement(lastNameLocator);
+        lastNameBox.sendKeys("happy");
+
+        Misc.pause(3);
+
+         // get the locator for email box and type
+        String emailXpath = "//input[@aria-label='Mobile number or email']";
+        By emailLocator = By.xpath(emailXpath);
+        WebElement emailBox = MyDriver.getDriver().findElement(emailLocator);
+        emailBox.sendKeys("jhjh@test.com");
+
+        Misc.pause(3);
+
+        // get the locator for re-enter email box and type
+        String re_EnteremailXpath = "//input[@aria-label='Re-enter email']";
+        By re_EnteremailLocator = By.xpath(re_EnteremailXpath);
+        WebElement re_EnteremailBox = MyDriver.getDriver().findElement(re_EnteremailLocator);
+        re_EnteremailBox.sendKeys("jhjh@test.com");
+
+        Misc.pause(3);
+
+        // get the locator for password box and type
+        String passwordXpath = "//input[@id='password_step_input']";
+        By passwordLocator = By.xpath(passwordXpath);
+        WebElement passwordBox = MyDriver.getDriver().findElement(passwordLocator);
+        passwordBox.sendKeys("12345");
+
+        Misc.pause(3);
+
+        // get the locator for sign up button and click it
+        String signUpXpath = "//button[@name='websubmit']";
+        By signUpLocator = By.xpath(signUpXpath);
+        WebElement signUpButton = MyDriver.getDriver().findElement(signUpLocator);
+        signUpButton.click();
+
+        Misc.pause(3);
+
+        // get the locator for "Please choose a gender. You can change who can see this later." text and verify its displayed
+        String chooseGenderXpath = "//div[contains(text(),'choose a gender.')]";
+        By chooseGenderLocator = By.xpath(chooseGenderXpath);
+        WebElement chooseGenderText = MyDriver.getDriver().findElement(chooseGenderLocator);
+        boolean isChooseGenderDisplayed = chooseGenderText.isDisplayed();
+        Assert.assertTrue(isChooseGenderDisplayed,"No is not displayed.");
+
+    }
+
+
+
+
```

Comment:
The second tests failed with this error: 

org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//div[contains(text(),'choose a gender.')]"}
  (Session info: chrome=100.0.4896.127)